src/System.IO.FileSystem/src/System/IO/Win32FileSystemEnumerable.cs

```diff
@@ -454,12 +456,32 @@ private void AddSearchableDirsToStack(SearchData localSearchData)
             }
         }
 
-        private static String NormalizeSearchPattern(String searchPattern)
+        // This is faster than using List<T>.Reverse because it avoids
+        // boxing in List<T>.Reverse's implementation due to the use of
+        // a custom struct for T.
+        private void ReverseSearchList(int index)
```

Comment:
I don't think we need to wait until the binaries make it to the relevant NuGet packages, but I think we should at least wait until the change is merged.